pull_and_redact_files.py

```diff
@@ -16,50 +16,60 @@ class DataRetriever:
           NETFILE_API_SECRET: the api secret for accessing NetFile API
         
         '''
-    
+
     def __init__(self, config, dest_dirpath='.local/netfile_redacted'):
         ''' Initialize with redaction configuration, destination directory and NetFile client '''
 
-        self.config = config
+        self.redaction_fields = config['redaction_fields']
+        self.endpoint_spec = config['endpoints']
         self.dest_dirpath = dest_dirpath
 
-        NETFILE_API_KEY = os.getenv('NETFILE_API_KEY','')
-        NETFILE_API_SECRET = os.getenv('NETFILE_API_SECRET','')
+        netfile_api_key = os.getenv('NETFILE_API_KEY','')
```

Comment:
If you prefer all variables to be the same, I'm ok with that.  So you know, the convention I was adopting was to keep all variables that are direct copies of an env var capitalized.  This way, it's obvious that it is the same as what is in the environment variable.  Then if I extract the string to another variable, the other variable is the conventional lower cased variable.

Reply:
That makes sense. This is another Pylint convention. It wants only constants to be UPPER_SNAKE, and it only considers a var a constant if it's in the global scope. I'm fine either way. I'm just always trying to get rid of those little red squiggles before I put up my code 😀

Reply:
That's good to know.  Maybe these variables should have been constants in the global scope.  I'll probably try to follow this convention going forward.